docs/samples/logger/local/README.md

```diff
@@ -0,0 +1,108 @@
+# Local Test of Kfserving Logger
+
+In one terminal start an echo http server on port 8000
+
+```
+docker run -it -p 8000:80 --rm -t mendhak/http-https-echo
+```
+
+Start an SKLearn Iris model on port 8081
+
+```
+python -m sklearnserver --model_dir gs://kfserving-samples/models/sklearn/iris --model_name sklearn-iris --http_port 8081
+```
+
+Start the Kfserving logger
+
+```
+../../../../bin/logger --log_url http://0.0.0.0:8000 --svc_port 8081 --sample 1.0 --log_mode all
+```
+
```

Comment:
The path really confuses me - can we make it more absolute from kfserving context?

Reply:
I could set an variable to make it clear. Just want the example to run if you are in the folder.